docset/winserver2012-ps/netsecurity/Set-NetFirewallAddressFilter.md

```diff
@@ -209,52 +194,36 @@ Accept wildcard characters: False
 
 ### -PolicyStore
 Targets the policy store from which to retrieve the rules to be modified. 
-  
 A policy store is a container for firewall and IPsec policy. 
 The acceptable values for this parameter are:
 
- -- PersistentStore: Sometimes called static rules, this store contains the persistent policy for the local computer.
+- PersistentStore: Sometimes called static rules, this store contains the persistent policy for the local computer.
 This policy is not from GPOs, and has been created manually or programmatically (during application installation) on the computer.
 Rules created in this store are attached to the ActiveStore and activated on the computer immediately. 
-
- -- ActiveStore: This store contains the currently active policy, which is the sum of all policy stores that apply to the computer.
+- ActiveStore: This store contains the currently active policy, which is the sum of all policy stores that apply to the computer.
 This is the resultant set of policy (RSOP) for the local computer (the sum of all GPOs that apply to the computer), and the local stores (the PersistentStore, the static Windows service hardening (WSH), and the configurable WSH). 
-
- ---- GPOs are also policy stores.
+---- GPOs are also policy stores.
 Computer GPOs can be specified as follows. 
-
- ------ `-PolicyStore hostname`. 
-
- ---- Active Directory GPOs can be specified as follows. 
-
- ------ `-PolicyStore domain.fqdn.com\GPO_Friendly_Namedomain.fqdn.comGPO_Friendly_Name`. 
-
- ------ Such as the following. 
-
- -------- `-PolicyStore localhost`
-
- -------- `-PolicyStore corp.contoso.com\FirewallPolicy`
-
- ---- Active Directory GPOs can be created using the New-GPO cmdlet or the Group Policy Management Console.
-  
-
- -- RSOP: This read-only store contains the sum of all GPOs applied to the local computer. 
-
- -- SystemDefaults: This read-only store contains the default state of firewall rules that ship with Windows Server® 2012. 
-
- -- StaticServiceStore: This read-only store contains all the service restrictions that ship with Windows Server 2012.
+------ `-PolicyStore hostname`. 
```

Comment:
It looks like this section didn't format correctly. All the text runs on a single line.

Reply:
This one is seriously messed up... I will try to fix it,